docs/Install/Creating_Task_Runners.md

```diff
@@ -0,0 +1,19 @@
+# <a name="GUID-CAD82E68-9A3A-465E-9DC2-06BBDCD1DC55"/> Creating Task Runners
+
+1.  In the **MyProjects**, click the **Project name** link.
+
+2.  In the **Task Runners** tab, click **Create new task runner**.
+
+    If you are running concurrent Swarm training, you need to create multiple task runners. If you are running a single training, the default task runner would be good enough and this step is optional.
```

Comment:
Add a note here "This feature is available for enterprise license users ONLY"

Reply:
Updated